packages/gui/src/constants.ts

```diff
@@ -201,6 +208,7 @@ export const defaultEnabledWidgets = {
   todo: true,
   weather: true,
   notes: true,
+  // markdown: true,
```

Comment:
Why not show the widget by default? I'ld suggest to have the [marquee docs for all widgets](https://github.com/stateful/vscode-marquee/blob/main/docs/CoreWidgets.md) as default configuration.
```suggestion
  markdown: true,
```